tests/test_controller.py

```diff
@@ -709,7 +721,7 @@ def test_nearby(self):
             eq_("NYPL", nypl['metadata']['title'])
             eq_("0 km.", nypl['metadata']['distance'])
             eq_("Connecticut State Library", ct['metadata']['title'])
-            eq_("35 km.", ct['metadata']['distance'])
+            eq_("29 km.", ct['metadata']['distance'])
```

Comment:
When we changed GeoIP libraries, we also changed which version of the database we're using, which changed the estimated coordinates assigned to IP addresses. This affects distance and any representations of latitude and longitude.

Reply:
That's unfortunate. Is this something to worry about for other data?

Reply:
I don't think so, the differences are approximately the distance between Manhattan and the Bronx. Not something that would affect the way we use the estimated locations.